spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -125,7 +125,7 @@ definitions:
         type: u8
         desc: |
           Quality of the STEC data. Encoded following RTCM DF389 specifcation
-          but as TECU instead of m.
+          but in units of TECU instead of m.
```

Comment:
I think it's some weird coded value that isn't TECU or meters.  still not quite how to interpret that spec...

Reply:
From the CLAS spec:
![image](https://user-images.githubusercontent.com/1215089/62508643-bded1500-b84b-11e9-903e-644ac8270f80.png)

From RTCM:
![image](https://user-images.githubusercontent.com/1215089/62508686-de1cd400-b84b-11e9-94f4-6b9aab8e500a.png)

In this case my reading is that they want you to use the SSR URA formula, but just replace metres with TECU, so it keeps the same numerical value, just interpreted differently.

I'm open to alternative wordings if you think there is something clearer we could use.